### PR TITLE
Adding RCA blog post

### PR DESCRIPTION
**Pull request by MasonEgger**

### CODE REVIEW

File: docs/blog/posts/2024-august-meetup-rca.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    - Incident Reports
+authors:
+    - masonegger
+---
+
+This post details an incident that occurred at the PyTexas Meetup on August 6, 2024.
+
+<!-- more -->
+
+On August 6, 2024, during the monthly PyTexas Meetup, a Code of Conduct
```
Comment (thejcannon) [inline docs/blog/posts/2024-august-meetup-rca.md]: Would it be fair to call it one event or many, or state it differently such that we simply the CoC was violated (making imthe quantity ambiguous)

File: docs/blog/posts/2024-august-meetup-rca.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+releasing this report.
+{.annotate}
+
+1. RCA = Root Cause Analysis
+
+## The Incident
+Below is a rough timeline of the events that transpired. As precise times as possible
+are given, although estimates are provided where we don't have exact times. 
+
+* *Between 7:00pm - 7:30pm*: Two unknown users joined the PyTexas
```
Comment (thejcannon) [inline docs/blog/posts/2024-august-meetup-rca.md]: "unknown" feels weird. Maybe "previously-unknown" or "new"?

File: docs/blog/posts/2024-august-meetup-rca.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+* *~7:30pm*: During the pre-meetup hangout in a Discord voice channel,
+    the above two users joined the channel. They then shared streamed their video,
+    which contained explicit pornographic material. The PyTexas Organizers immediately 
+    reacted by banning the users from the server. After the incident, the pre-meetup
+    hangout continued.
+* *8:01pm*: The Meetup starts on time in the Meetup stage in Discord.
+* *8:02pm*: Links to the monthly attendance survey and Slido were
+    distributed in the text chat.
+* *8:06pm*: An emoji appeared on a message in the stage chat very
+    briefly. The emoji was an antisemitic symbol. It appeared and was immediately
```
Comment (thejcannon) [inline docs/blog/posts/2024-august-meetup-rca.md:38]: Did we remove it or Discord?

File: docs/blog/posts/2024-august-meetup-rca.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+for users so the option isn't available for anyone who joins, but members would 
+have to earn the permission.
+
+### Slido not providing moderation tooling in its free tier
+Slido moderation tools only exist at a high tier of their product. This is honestly,
+annoying. Basic moderation tools shouldn't be behind a pay wall. We used Slido 
+at the 2024 PyTexas Conference, but had purchased a multi-day premium plan. 
+This is not cheap, and currently is not an option for us financially.
+
+**Corrective Action** - We are currently moving off of Slido to Google Forms.
```
Comment (thejcannon) [inline docs/blog/posts/2024-august-meetup-rca.md:120]: Maybe worth mentioning something along the lines ofhow this is a temporary fix?

Comment (thejcannon) [reply docs/blog/posts/2024-august-meetup-rca.md:120]: (e g. Link to the below?)

**Review by thejcannon — APPROVED** [review]
It's good we're doing this. I can't support it enough

File: docs/blog/posts/2024-august-meetup-rca.md (hunk 1 above)
Comment (MasonEgger) [reply docs/blog/posts/2024-august-meetup-rca.md]: Changing to `On August 6, 2024, during the monthly PyTexas Meetup, an incident involving Code of Conduct violations occurred.`

File: docs/blog/posts/2024-august-meetup-rca.md (hunk 3 above)
Comment (MasonEgger) [reply docs/blog/posts/2024-august-meetup-rca.md:38]:
```
The emoji was an antisemitic symbol. It appeared and was immediately removed. We are unsure who removed it, either the poster or Discord's auto-moderation tooling.
```

File: docs/blog/posts/2024-august-meetup-rca.md (hunk 4 above)
Comment (MasonEgger) [reply docs/blog/posts/2024-august-meetup-rca.md:120]: I think it's fine as it. If they're interested they'll read below.

**Pull request merged**